.github/workflows/Push.yml

```diff
@@ -0,0 +1,31 @@
+name: Master Push
+
+on:
+  push:
+    branches: [master]
+
+jobs:
+  format:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v2
+        with:
+          token: ${{ secrets.PAT }}
+
+      - uses: actions/setup-node@v2
+        with:
+          node-version: '16'
+
+      - uses: c-hive/gha-npm-cache@v1
+
+      - name: Install dependencies
+        run: npm ci
+
+      - name: Format
+        run: npm run format
+
+      - name: Commit formatting changes
+        uses: stefanzweifel/git-auto-commit-action@v4
+        with:
+          commit_message: Apply formatting changes
+          branch: ${{ github.head_ref }}
```

Comment:
A new line at the end of the file is missing.

Reply:
Why do we need an empty new line? Either way when code is merged the format script will run and format the code when it's in master in the way it is configured to in the `.prettierrc`

Reply:
Talking about the prettier config, I will make some updates to that because it currently just has one rule 🤨. Will push that here later too!

Reply:
Regarding the prettier configuration, I think it was your preference to avoid trailing commas, which is different from the default.

Reply:
Yeah but we can do more, I will update it in a different PR actually

Reply:
Updated the prettier config in PR #102